docs/getting_started.rst

```diff
@@ -151,3 +151,69 @@ we can export books::
     Data exported programmatically is not sanitized for malicious content.
     You will need to understand the implications of this and handle accordingly.
     See :ref:`admin_security`.
+
+Using modelresource_factory
```

Comment:
Please could you move this to `advanced_usage.rst`?

Reply:
> Please could you move this to advanced_usage.rst?

Agree, keep only the `modelresource_factory` link in getting_started.rst and moved docs to advanced_usage.rst.(commit: a1c2487807854501c2ff78d034fe99da7ea28777)

@matthewhegarty